lib/modelinstance.js

```diff
@@ -568,20 +568,25 @@ ModelInstance.prototype.load = function(callback) {
   }
 
   if (!this.loaded()) {
-    var key = _modelKey(self);
-    $.schema.store.get(key, function(err, data, cas) {
-      if (err) {
-        finalCallback(err);
-        return;
-      }
-
-      var md = new ModelData();
-      md.key = $.key;
-      md.data = data;
-      md.cas = cas;
-      ModelInstance.applyData(self, md);
-      loadSubItem();
-    });
+      $.schema.execPreHandlers('load', self, function () {
```

Comment:
Previously, load handlers weren't called at all, now they are; but only in the code path where THIS model is loaded.  If sub-models are loaded, load signal for those is delegated to those models.